### PR TITLE
refactor(esx/client/functions): rewrite spawnobject function

### PR DESCRIPTION
**Pull request by Thekuca**

### CODE REVIEW

File: [core]/es_extended/client/functions.lua [hunk 1]
```diff
@@ -356,36 +356,35 @@ function ESX.Game.Teleport(entity, coords, cb)
...
-                cb(obj)
-            end
-        end)
-    end
+---@param object number|string
+---@param coords vector3|table
+---@param networked boolean
+---@param cb function
+function ESX.Game.SpawnObject(object, coords, networked, cb)
+   	local networked = networked ~= false
```
Comment (Gellipapa) [inline [core]/es_extended/client/functions.lua:364]: networked nil?

Comment (Thekuca) [reply [core]/es_extended/client/functions.lua:364]: nil makes true

**Comment by Gellipapa** [comment]
@Thekuca Why close this pr?

**Comment by Thekuca** [comment]
> @Thekuca Why close this pr?

fantik changed plans

**Pull request closed without merging**